modules/migration-excluding-resources.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc
+// * migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc
+[id='excluding-resources-from-migration-plan_{context}']
+= Excluding resources from a migration
```

Comment:
You shouldn't have multiple procedures in the same module like this. Since the procedure is so similar for the different methods, I would merge the three sets of steps into one. I'd convert the step where you modify the Migration Controller CR to substeps and explain the three different modifications. I'd probably show command output that assumes that you made all the modifications possible.

Reply:
Good idea. I was trying to figure out how to combine all these into one procedure.